features/active-view-transition.yml.dist

```diff
@@ -0,0 +1,28 @@
+# Generated from: active-view-transition.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "125"
+    chrome_android: "125"
+    edge: "125"
+compat_features:
+  # baseline: false
+  # support:
+  #   chrome: "125"
+  #   chrome_android: "125"
+  #   edge: "125"
+  #   safari: "18"
+  #   safari_ios: "18"
+  - css.selectors.active-view-transition
+
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: false
+  # support:
+  #   chrome: "125"
+  #   chrome_android: "125"
+  #   edge: "125"
+  - api.ViewTransition.types
+  - api.ViewTransitionTypeSet
+  - css.selectors.active-view-transition-type
```

Comment:
There's a chance that someone in review (or down the road) might want to refer to `active-view-transition` and `active-view-transition-type` separately, but I think keep them combined until someone suggests otherwise. It could really go either way.